tests/MCNPinput_test.py

```
@@ -808,8 +809,11 @@ def test_get_reaction_file(self):
 
         lib = "99c"
         reacfile = newinp.get_reaction_file(self.lm, lib)
-        assert ["24050", "78195"] == reacfile.get_parents()
-        reacfile.reactions[1].daughter == "78195900"
+        assert [
+            Nuclide.from_int_string("24050.99c"),
+            Nuclide.from_int_string("78195.99c"),
+        ] == reacfile.get_parents()
+        reacfile.reactions[1].daughter.write_to_int_string() == "78195900"
```

Ruff (0.14.8)

Pointless comparison at end of function scope. Did you mean to return the expression result?

(B015)